### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by lvaleriu** [keunwoochoi/MSD_split_for_tagging#4]
What loss, accuracy you obtain when training with the compact_cnn model? I'm using the default parameters and loading model like this: model = main('tagger'). But I get a huge overfit very quick. Should i be more patient? How many epochs?

Are you doing binary or categorical classification? It seems to me that it is binary_classification but I want to be sure.

**Comment by lvaleriu** [comment]
Using TensorFlow backend.
2017/10/17
Keras version: 2.0.8
Keras backend: tensorflow: 1.3.0
Keras image data format: channels_first
Kapre version: 0.1.2.1

Layer (type)                 Output Shape              Param #   
========================================
melgram (Melspectrogram)     (None, 1, 96, 1360)       287840    
_________________________________________________________________
ConvBNEluDr (Sequential)     (None, 32, 1, 1)          37952     
_________________________________________________________________
flatten_1 (Flatten)          (None, 32)                0         
_________________________________________________________________
dense_1 (Dense)              (None, 50)                1650      
=========================================
Total params: 327,442
Trainable params: 39,282
Non-trainable params: 288,160
_________________________________________________________________
Epoch 1/1000
256/256 [==============================] - 619s - loss: 5.9552 - acc: 0.2142 - val_loss: 7.7083 - val_acc: 0.0879
Epoch 2/1000
256/256 [==============================] - 586s - loss: 5.5266 - acc: 0.2606 - val_loss: 7.7248 - val_acc: 0.0803
Epoch 3/1000
256/256 [==============================] - 577s - loss: 5.2884 - acc: 0.2843 - val_loss: 5.4751 - val_acc: 0.2380
Epoch 4/1000
256/256 [==============================] - 576s - loss: 5.2836 - acc: 0.2948 - val_loss: 7.0165 - val_acc: 0.0959
Epoch 5/1000
